gatsby-config.mjs

```diff
@@ -39,10 +41,14 @@ const skipWebpackAnalyzer = has(process.env, 'GITHUB_ACTIONS') || has(process.en
 const getContentfulHost = () => {
   if (has(process.env, 'GITHUB_ACTIONS')) return 'cdn.contentful.com';
 
-  // This is an environment variable set by the Netlify build process.
-  const netlifySiteName = get(process.env, 'SITE_NAME');
-  if (netlifySiteName === 'roadie-preview') return 'preview.contentful.com';
-  if (netlifySiteName === 'roadie') return 'cdn.contentful.com';
+  // This environment variable is always set when building in the netlify environment. It is
+  // not set by the netlify dev server.
+  if (get(process.env, 'NETLIFY')) {
```

Comment:
The presence of the Netlify `SITE_NAME` env var was being used to check if we are executing on netlify (aka a production like environment). However, this broke use of the netlify dev server `yarn netlify dev` because that also sets the `SITE_NAME` env var. To make this more explicit, I added a check for the `NETLIFY` env var, whcih is only present when we are executing on Netlify.